lib/cosmos/tools/packet_viewer/packet_viewer.rb

```diff
@@ -411,15 +408,16 @@ def update_tlm_items(featured_item_name = nil)
 
             begin
               tlm_items = get_tlm_packet(target_name || '', packet_name || '', @mode)
-            rescue DRb::DRbConnError => err
+            rescue DRb::DRbConnError => error
               Qt.execute_in_main_thread(true) do
                 statusBar.showMessage(tr("Error Connecting to Command and Telemetry Server"))
               end
               tlm_items = nil
               update_needed = false
-            rescue RuntimeError => err
+            rescue RuntimeError => error
               Qt.execute_in_main_thread(true) do
-                statusBar.showMessage(tr("Packet #{target_name} #{packet_name} does not exist"))
+                Cosmos.handle_critical_exception(error)
```

Comment:
I think a dialog is needed here or you have no idea why Packet Viewer just didn't display your packet. The stack backtrace clearly identifies where the error occured (typically your custom conversion code).